.github/workflows/lint-build.yml

```diff
@@ -9,12 +9,14 @@ jobs:
         with:
           node-version: '18'
           cache: 'yarn'
-      - name: Set up MySQL
+      - name: Set up PostgreSQL
         run: |
-          sudo /etc/init.d/mysql start
-          mysql -e 'CREATE DATABASE checkpoint;' -uroot -proot
+          sudo systemctl start postgresql
+          sudo -u postgres psql -c "CREATE DATABASE checkpoint;"
+          sudo -u postgres psql -c "CREATE USER checkpoint_user WITH PASSWORD 'checkpoint_pass';"
+          sudo -u postgres psql -c "ALTER DATABASE checkpoint OWNER TO checkpoint_user;"
       - run: yarn install
       - run: yarn lint
       - run: yarn build
         env:
-          DATABASE_URL: 'mysql://root:root@l27.0.0.1:3306/checkpoint'
+          DATABASE_URL: 'postgresql://checkpoint_user:checkpoint_pass@localhost:5432/checkpoint'
```

Comment:
Can you add final blank line there?

Reply:
58867b70d736330cb60d4faef8f07c17616ba62e